roles/servicetelemetry/tasks/_local_signing_authority.yml

```diff
@@ -5,8 +5,8 @@
       apiVersion: cert-manager.io/v1
       kind: Issuer
       metadata:
-        name: '{{ meta.namespace }}-selfsigned'
-        namespace: '{{ meta.namespace }}'
+        name: '{{ ansible_operator_meta.namespace }}-selfsigned'
```

Comment:
I'm assuming that this is a change due to the var being renamed in the newer operator version